test/ngSanitize/sanitizeSpec.js

```diff
@@ -246,25 +246,28 @@ describe('HTML', function() {
       .toEqual('<p>text1text2</p>');
   });
 
-  it('should throw on clobbered elements', function() {
-    inject(function($sanitize) {
-      expect(function() {
-        $sanitize('<form><input name="parentNode" /></form>');
-      }).toThrowMinErr('$sanitize', 'elclob');
+  if (!/Edge\/16/.test(window.navigator.userAgent)) {
+    // Skip test on Edge 16 due to browser bug.
```

Comment:
Perhaps include this link here: https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/15263856/#pagenotfound

Reply:
Problem is that the issue is private, so it goes to 404. Maybe we should create an issue in our repo with a short description?